subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -112,7 +112,7 @@ class AndroidSantaTrackerSmokeTest extends AbstractAndroidSantaTrackerSmokeTest
         result = runnerForLocationExpectingLintDeprecations(checkoutDir, agpVersion, "lintDebug",
             [
                 "wearable-2.3.0.jar (com.google.android.wearable:wearable:2.3.0)",
-                "kotlin-android-extensions-runtime-1.5.10.jar (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.5.10)",
+                "kotlin-android-extensions-runtime-1.5.21.jar (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.5.21)",
```

Comment:
Should we use the Kotlin version we pass to the santa tracker project here, or is this version different?

Reply:
Good idea, reusing it now.